extension/intents/notes/notes.js

```diff
@@ -82,7 +82,14 @@ intentRunner.registerIntent({
       e.displayMessage = "You have not set a tab to write";
       throw e;
     }
-    await browserUtil.makeTabActive(writingTabId);
+    const available = await content.hasScript(writingTabId, SCRIPT);
```

Comment:
I _just_ removed this function yesterday. But really it's enough to just catch the error from `browserUtil.makeTabActive()` – if it raises an exception about an invalid tab ID then the tab is gone. So it's a matter of catching that exception and adding `.displayMessage` to it.

Reply:
@ianb I made the changes as you asked, Thanks for the suggestion :)